src/AstroBase.jl

```diff
@@ -81,16 +81,15 @@ function xy06(jd1, jd2)
            end
         end
 
-        sc[1] = sin(arg)
-        sc[2] = cos(arg)
+        sc[2], sc[1] = reim(cis(arg))
```

Comment:
I literally meant `sincos`.  `reim(cis(x))` in Julia 0.6 is just the same as `sin(x), cos(x)`, with added overhead of passing through complex numbers.

I now see that `sincos` is not available in Julia 0.6.  In AstroLib.jl I added this compat definition: https://github.com/JuliaAstro/AstroLib.jl/blob/73621b179b818464e72e942fd3a691ee79f14243/src/common.jl#L180

Reply:
We might want to start targeting 0.7.